articles/service-fabric/service-fabric-backup-restore-service-trigger-restore.md

```diff
@@ -160,22 +168,31 @@ For _Named Partitioning_, the name value is compared to identify the target part
 
 #### Powershell using Microsoft.ServiceFabric.Powershell.Http Module
 
+Replace `account-name` with your storage account name.
+
 ```powershell
 
-Restore-SFPartition  -PartitionId '1c42c47f-439e-4e09-98b9-88b8f60800c6' -BackupId 'b0035075-b327-41a5-a58f-3ea94b68faa4' -BackupLocation 'SampleApp\MyStatefulService\974bd92a-b395-4631-8a7f-53bd4ae9cf22\2018-04-06 21.10.27.zip' -AzureBlobStore -ConnectionString 'DefaultEndpointsProtocol=https;AccountName=<account-name>;AccountKey=<account-key>;EndpointSuffix=core.windows.net' -ContainerName 'backup-container'
+Restore-SFPartition -PartitionId 1c42c47f-439e-4e09-98b9-88b8f60800c6 -BackupId b0035075-b327-41a5-a58f-3ea94b68faa4 -BackupLocation 'SampleApp\MyStatefulService\974bd92a-b395-4631-8a7f-53bd4ae9cf22\2018-04-06 21.10.27.zip' -ManagedIdentityAzureBlobStore -BlobServiceUri "https://<account-name>.blob.core.windows.net" -ContainerName "backup-container" -ManagedIdentityType "VMSS" -FriendlyName "AzureMI_storagesample" ManagedIdentityClientId "<clien-ID of User-Assigned MI>" 
+
+# Use Optional parameter `ManagedIdentityClientId` with clien-ID of User-Assigned Managed Identity in case of multiple User-Assigned managed identities assigned to your resource, else no need of this paramter.
```

Comment:
Same needs to be clear when to use ManagedIdentityClientId

Reply:
updated